api/src/main/java/org/eclipse/microprofile/config/Config.java

```diff
@@ -69,13 +69,11 @@
      *
      * @param propertyName
      *             The configuration propertyName.
-     * @return The resolved property value as a String-Optional, which will bypass converters.
+     * @return The resolved property value as a String, which will bypass any converters.
      *
-     * @throws IllegalArgumentException
-     *             is thrown if the propertyName maps to an object that is not a
-     *             String.
+     * 
      */
-    Optional<String> getString(String propertyName);
+    String getRawString(String propertyName);
```

Comment:
I assume that the method is intended for debugging rather than for business logic, therefore it is counter-productive to wrap the result in an Optional. It would make even sense to me to rename the method to something much more complicated to discourage using the method in normal business logic :) But I'm serious - far too many people would use it for normal code if its name stays like this.